### DISCUSSION TITLE
Add Fantom Icon to assets/chains.tsx and display it when the chain is selected

### DISCUSSION THREAD

**Discussion by nikmace** [family/connectkit#140] [Ideas]
It would be nice to **add support for more chains** to have an icon instead of just showing a default one.

API Reference for Chain Icon: https://docs.family.co/connectkit/api-reference#additional-components-chain-icon

Would it be possible to add Fantom to the 'list'? Link to Fantom icon, just in case: https://cryptologos.cc/fantom. Thank you in advance! :)

**Comment by lochie** [comment]
have drafted PR https://github.com/family/connectkit/discussions/140 which adds some more chain logos, including Fantom 😊

**Reply by nikmace** [reply]
Thank you very much, appreciate it! :)
